installer/helm/chart/volcano/values.yaml

```diff
@@ -147,3 +147,19 @@ custom:
   admission_log_level: 4
   scheduler_log_level: 3
   controller_log_level: 4
+
+# Specify container security context for admission
+# For example:
+#
+# admission_default_csc:
+#   allowPrivilegeEscalation: false
+#   runAsUser: 2000
+  admission_default_csc: ~
+
+# Specify container security context for scheduler
```

Comment:
We can put these `admission_default_csc: ~` and `scheduler_default_csc: ~` together.

Reply:
You mean using the same definition for both ?

Reply:
forgot about it, I saw how other fields were made so I'll follow the same behavior

Reply:
Done

Reply:
Ok, and please also squash it to one commit.